packages/link/test/link.test.ts

```diff
@@ -46,4 +46,20 @@ describe('Link', () => {
 
         await expect(el).to.be.accessible();
     });
+
+    it('loads[rel]', async () => {
+        const el = await fixture<Link>(
+            html`
+                <sp-link href="test_url" rel="external">
+                    Default Link
+                </sp-link>
+            `
+        );
+
+        await elementUpdated(el);
+        expect(el).to.not.be.undefined;
+        expect(el.getAttribute('rel')).to.eq('external'));
+
+        await expect(el).to.be.accessible();
```

Comment:
```suggestion
        expect(el.focusElement.getAttribute('rel')).to.eq('external'));
        await expect(el).to.be.accessible();
```

Looks like the tests here may have been over simplified in the most recent update and aren't laying out the best example to work from here. This will confirm the `rel` attribute is being passed into the `<a>` in the shadow DOM as expected.

Reply:
resolved by 2f85823